theano/sandbox/rng_mrg.py

```diff
@@ -371,9 +376,11 @@ def c_code(self, node, name, inp, out, sub):
         assert isinstance(node.inputs[0].type, TensorType)
         o_rstate, o_sample = out
         if self.inplace:
-            o_rstate_requirement = 'NPY_ARRAY_C_CONTIGUOUS|NPY_ARRAY_ALIGNED'
+            o_rstate_requirement = \
```

Comment:
It is preferable to avoid '\' as much as possible. Use parenthesis instead, like:
```python
o_rstate_requirement = (
    'NPY_ARRAY_C_CONTIGUOUS|NPY_ARRAY_ALIGNED')
```